parm/config/gfs/config.resources.JET

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env bash
+
+# Jet-specific job resources
+
+case ${PARTITION_BATCH} in
+  "xjet") node_mem="61GB";;
+  "vjet") node_mem="61GB";;
+  "sjet") node_mem="29GB";;
+  "kjet") node_mem="93GB";;
+  *)
+    echo "FATAL ERROR: Unknown partition ${PARTITION_BATCH} specified for Jet"
+    exit 3
+esac
```

Comment:
Should this be in `config.resources` (case statement block of line 43) and be called `node_mem_max`?

Reply:
I think I like that solution better and to declare it for each system, even though it's not used for other systems (yet).